book/src/recursive/coinduction.md

```diff
@@ -1,3 +1,73 @@
 # Coinduction
 
-TBD
\ No newline at end of file
+This sub-chapter is meant to describe the current handling of coinductive goals in the recursive solver rather than providing an extensive overview over the theoretical backgrounds and ideas.
+It follows the description in [this GitHub comment](https://github.com/rust-lang/chalk/issues/399#issuecomment-643420016) and the Zulip topic linked there.
+
+## General Idea
+The general idea for the handling of coinductive cycles in the recursive solver is to start by assuming the goal is provable and then try to find evidence that it is not.
+This search for a disproof is done by the standard recursive solving process described in the sub-chapters before.
+
+Albeit this approach would allow for the handling of mixed inductive/co-inductive cycles, these are actually handled as errors to prevent propagation of the assumed provability outside of the coinductive cycle.
```

Comment:
See my explanation below. Maybe we can add a subsection like

## Mixed co-inductive and inductive cycles

and then reference that section from the code? My comment seems to long to include in the code itself.

Reply:
I tried adding a comprehensive subsection about mixed cycles. I hope that it is understandable.